spec/forms/dependant_income_details_form_spec.rb

```diff
@@ -48,4 +48,78 @@
       expect(page).not_to have_text "Dependant 3"
     end
   end
+
+  describe "dependant income limits" do
+    context "when choosing monthly frequency" do
+      let(:monthly_limit) { "338.90" }
+
+      it "shows an error message when on the limit" do
+        fill_in "1-amount", with: monthly_limit
+        choose "1-frequency-monthly"
+        click_on "Save and continue"
+        expect(page).to have_content "Dependant income cannot be more than £338.90 per month."
```

Comment:
This to me looks incorrect. If the income is 338.90, an error message saying it cannot be _more_ than 338.90 doesn't seem like it applies. I suspect that we should only show the message if dependant income is 338.91 or above?

Reply:
I was going off the original A/C which I can see has just been changed, so will update accordingly.